walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/script.js

```diff
@@ -30,3 +30,9 @@ function randomizeImage() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+function init() {
+  // document.getElementById("content").style.color = 'green';
```

Comment:
Nit: Please remove commented code. You can always use version history to get it back

Reply:
Ohh that slipped, sorry will remove now!